gdpr/aggregated_config.go

```diff
@@ -50,6 +51,13 @@ func (tc *tcf2Config) IntegrationEnabled(integrationType config.IntegrationType)
 	return tc.HostConfig.Enabled
 }
 
+func (tc *tcf2Config) ChannelEnabled(integrationType config.IntegrationType) bool {
```

Comment:
Please use the comment for the function `IntegrationEnabled` just above this new function and alter it accordingly.